### PR TITLE
Snippets 04 - Part 2

### PR DESCRIPTION
**Pull request by qsm-odoo**

### CODE REVIEW

File: addons/website/static/src/snippets/s_blockquote/001.scss [hunk 1]
```diff
@@ -36,8 +36,11 @@
     // Cover
     &.s_blockquote_cover {
         text-align: center;
-        .s_blockquote_icon + .s_blockquote_content {
-            margin-top: -$spacer * 1.5;
+        .s_blockquote_icon {
+            float: none !important;
```
Comment (qsm-odoo) [inline addons/website/static/src/snippets/s_blockquote/001.scss:40]: To avoid the important rule, can't you remove the float class from the dom and have a float rule for the non-cover style instead?

File: addons/website/views/snippets/s_popup.xml [hunk 2]
```diff
@@ -34,7 +35,7 @@
 
 <template id="s_popup_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-js="SnippetPopup" data-selector=".s_popup" data-target=".modal">
+        <div data-js="SnippetPopup" data-selector=".s_popup:not('#website_cookies_bar')" data-target=".modal">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/s_popup.xml:38]: Not sure why? The whole point of using s_popup structure for the cookie bar was to have the option working on the cookie bar too :)

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss [hunk 3]
```diff
@@ -8,16 +8,15 @@
         display: none;
     }
     .close {
+        z-index: $zindex-modal;
```
Comment (qsm-odoo) [inline addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:11]: Why is this needed?

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss [hunk 4]
```diff
@@ -8,16 +8,15 @@
...
         @include o-position-absolute(0, 0);
-        font-size: $font-size-sm;
-        width: 2 * $font-size-sm;
-        height: 2 * $font-size-sm;
+        height: $font-size-lg * 2;
+        width: $font-size-lg * 2;
```
Comment (qsm-odoo) [inline addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:14]: width before height to control my allergies please :p

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss [hunk 5]
```diff
@@ -8,16 +8,15 @@
...
+        line-height: $font-size-lg * 2;
+        margin: 0;
         padding: 0;
-        margin: (-$font-size-sm) (-$font-size-sm) 0 0;
-        opacity: 1;
-        background-color: $white;
-        z-index: 1;
-        border-radius: $font-size-sm;
+        @include o-bg-color(color-yiq(o-color('primary')), o-color('primary'));
```
Comment (qsm-odoo) [inline addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:18]: Add the parameter `$with-extras: false` otherwise this generate useless extra rules ;) ... maybe I should change the default of to be false someday... :)

File: addons/website_mass_mailing/views/snippets_templates.xml [hunk 6]
```diff
@@ -54,11 +54,26 @@
...
+    </section>
+    <section>
+        <div class="container">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-8 offset-lg-2 pt32 pb32">
+                    <t t-snippet-call="website_mass_mailing.s_newsletter_subscribe_form"/>
+                </div>
+            </div>
+        </div>
+    </section>
```
Comment (qsm-odoo) [inline addons/website_mass_mailing/views/snippets_templates.xml:72]: Mmh, the point of using a base snippet for that popup was that you could re-add it if you removed it from the popup... but I guess the user can just CTRL-Z or redrop the popup snippet. Should we remove the s_newsletter_block snippet though? cc @sbuhl 

However, if we do that, the `section` elements here should be standard snippet (so appropriate s_ classes and data-snippet attribute), otherwise we cannot deprecate them ever ;)

**Comment by qsm-odoo** [comment]
You can also rebase the branch, I think you will have no conflict and the first commit will disappear (if not, I can do the rebase once I test if you want)

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss (hunk 3 above)
Comment (Cocographique) [reply addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:11]: Otherwise, in some situations, it can disappear behind the content.

Screenshot 2020-06-11 at 17 46 50

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss (hunk 4 above)
Comment (Cocographique) [reply addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:14]: This mistake could have been avoided by using the size mixin 😜

File: addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss (hunk 5 above)
Comment (Cocographique) [reply addons/website_mass_mailing/static/src/scss/website_mass_mailing_popup.scss:18]: 👍

**Comment by qsm-odoo** [comment]
@robodoo rebase-ff r+
